### PR TITLE
FIX:Removed Receipt.txt in the Billing and transaction curated view

### PR DESCRIPTION
**Pull request by NKannan14**
removed receipt.txt from billing and transaction curated view
Removed standardised lambda test as we have end to end test for invoice 
Increased timeout in invoice end to end test

### CODE REVIEW

**Review by Si3rr4wow — APPROVED** [review]
good bye file

**Comment by MelAtGDS** [comment]
So timeout has been increased for the `s3-standardised-lambda-test` which now passed but the new failing test is in `s3-invoice-raw-store-test` for the **Unhappy path - Upload invalid pdf to the raw invoice pdf bucket test**.

**Review by markschnitziusgovuk — APPROVED** [review]

File: integration_tests/tests/billing-and-transaction-view-tests.ts [hunk 1]
```diff
@@ -9,56 +11,47 @@ import {
...
-  const folderPrefix = "btm_billing_standardised";
-  const testObject: S3Object = {
-    bucket: `${prefix}-storage`,
-    key: `${folderPrefix}/receipt.txt`,
-  };
-  beforeAll(async () => {
+  beforeEach(async () => {
     // tests are enabled to run sequentially as we are deleting the S3 directory in view tests so when running the test
     // in parallel other tests will be interrupted(e.g. sns-s3 tests generate and checks eventId). We can enable to run in parallel
     // once we implement BTM-340 to clean up after each test
```
Comment (MelAtGDS) [inline integration_tests/tests/billing-and-transaction-view-tests.ts:36]: I can't find BTM-340 in Jira so I don't remember what it is was for and if these comments need to be updated.

File: integration_tests/tests/billing-and-transaction-view-tests.ts [hunk 2]
```diff
@@ -84,22 +77,79 @@ interface TestData {
...
+          s3Object.LastModified !== undefined &&
+          new Date(s3Object.LastModified) >= testStartTime
+      ),
+    {
+      timeout: 50000,
+      nonCompleteErrorMessage:
+        "Invoice data never appeared in standardised folder",
+    }
+  );
+  console.log("**********", res);
```
Comment (MelAtGDS) [inline integration_tests/tests/billing-and-transaction-view-tests.ts:146]: I think perhaps these three console.logs can be removed now.

File: integration_tests/tests/billing-and-transaction-view-tests.ts (hunk 1 above)
Comment (NKannan14) [reply integration_tests/tests/billing-and-transaction-view-tests.ts:36]: this is the ticket to improve the cleanup strategy https://govukverify.atlassian.net/browse/BTM-340

File: integration_tests/tests/billing-and-transaction-view-tests.ts (hunk 2 above)
Comment (NKannan14) [reply integration_tests/tests/billing-and-transaction-view-tests.ts]: done

**Review by MelAtGDS — APPROVED** [review]

**Pull request merged**